README.md

```diff
@@ -446,7 +446,7 @@ Services to securely store your Docker images.
 * [AMP](https://github.com/appcelerator/amp) - The open source unified CaaS/FaaS platform for Docker, batteries included. By [@Appcelerator](https://github.com/appcelerator/)
 * [Apache OpenWhisk](https://github.com/apache/incubator-openwhisk) - a serverless, open source cloud platform that executes functions in response to events at any scale. By [@apache](https://github.com/apache)
 * [Docker-Lambda](https://github.com/lambci/docker-lambda) - Docker images and test runners that replicate the live AWS Lambda environment. By [@lamb-ci](https://github.com/lambci)
-* [FaaS](https://github.com/openfaas/faas) - Docker Serverless/Functions as a Service. By [@openfaas](https://github.com/openfaas/)
+* [OpenFaaS](https://github.com/openfaas/faas) - A complete serverless functions framework for Docker and Kubernetes. Twitter: [@open_faas](https://github.com/open_faas/), Web: [openfaas.com](https://www.openfaas.com)
```

Comment:
@alexellis thanks for your contribution. Actually, we prefer GitHub link only in this case. Please also list the tool in alphabetical order.